test/expected/service-metadata-best-effort.json

```diff
@@ -0,0 +1,36 @@
+[
+    {
+        "lang": "en",
+        "meta": {
+            "description": "Census 2021 metadata",
+            "build": {
```

Comment:
Maybe a little late in the day to raise this thought, but is `build` safe to use and unlikely to clash with any other properties that might be needed by ONS? My only other suggestion would be `_build` - but I'm thinking we are probably unlikely to have a clash, what do you think?

Reply:
We don't have any ONS specific information at the service level. I think we should be fine with `build`.